unlock-app/src/components/content/event/LandingPage.tsx

```diff
@@ -0,0 +1,226 @@
+          src="/images/illustrations/events/party.svg"
+        />
+      </div>
+
+      <section className="absolute left-0 justify-items-center justify-center content-center items-center	text-white bg-black w-screen	flex flex-col py-8">
+        <h1 className="text-xl font-semibold">Used by</h1>
+        <ul className="flex flex-row my-8">
+          <li className="md:mx-12 mx-2 flex items-center rounded-full text-center w-24 h-24">
+            <Link target="_blank" href="https://ethcc.io/">
+              <Image
+                width="100"
+                height="100"
+                alt="ethcc"
+                src="/images/illustrations/events/ethcc.svg"
+              ></Image>
+              <h4 className="mt-auto">EthCC</h4>
+            </Link>
+          </li>
+          <li className="md:mx-12 mx-2 flex items-center rounded-full text-center w-24 h-24 ">
+            <Link target="_blank" href="https://www.dappcon.io/">
+              <Image
+                width="100"
+                height="100"
+                alt="dappcon"
+                src="/images/illustrations/events/dappcon.png"
+              ></Image>
+              <h4 className="mt-auto">DappCon</h4>
+            </Link>
+          </li>
+          <li className="md:mx-12 mx-2 flex items-center rounded-full text-center w-24 h-24">
+            <Link target="_blank" href="https://www.metacartel.org/">
+              <Image
+                width="100"
+                height="100"
+                alt="metacartel"
+                src="/images/illustrations/events/metacartel.png"
+              ></Image>
+              <h4 className="mt-auto">Metacartel</h4>
+            </Link>
+          </li>
+        </ul>
+      </section>
+      <section className="flex flex-col mt-96 justify-items-center	justify-center content-center items-center">
+        <h3 className="text-5xl font-semibold text-center md:w-2/3">
+          Ticketing events with Unlock is simple.
+        </h3>
+        <p className="text-center md:w-1/2 mt-4">
+          From creating the event description to selling tickets to check-in,
+          Unlock built the tools you need.
+        </p>
+        <ul className="grid md:grid-cols-[300px_300px_300px] gap-8 my-8">
+          <li className="bg-red">
+            <Image
+              className="border border-1 rounded-lg border-ui-main-100"
+              width="400"
+              height="300"
+              alt="No-code"
+              src="/images/illustrations/events/easy.svg"
+            ></Image>
+            <h3 className="text-xl font-bold mb-2 mt-2">
+              No-code smart contract deployment
+            </h3>
+            <p className="text-sm">
+              Simply fill up the form and hit the deploy button. All the
+              metadata information will automatically be included in the NFT
+              tickets. You can always add and modify other properties at later
+              date.
+            </p>
+          </li>
```

Comment:
Same here. Can you extract the data outside if it's not too much work so it's easier to modify?

Reply:
YEs